jedi_vim.py

```diff
@@ -419,7 +422,20 @@ def show_call_signatures(signatures=()):
         tup = '%s, %s' % (len(add), replace)
```

Comment:
The `tup` variable and its use from `regex` can be removed. Previously this was used to restore the original line, but it's now not used any more.